ipv-api/src/main/java/uk/gov/di/authentication/ipv/lambda/IPVCallbackHandler.java

```diff
@@ -212,21 +199,44 @@ public APIGatewayProxyResponseEvent handleRequest(
                                     ipvAuthorisationService.validateResponse(
                                             input.getQueryStringParameters(),
                                             session.getSessionId()));
-            if (errorObject.isPresent()) {
-                return generateAuthenticationErrorResponse(
-                        authRequest,
-                        new ErrorObject(ACCESS_DENIED_CODE, errorObject.get().getDescription()),
-                        false,
-                        clientSessionId,
-                        session.getSessionId());
-            }
             var userProfile =
                     dynamoService
                             .getUserProfileFromEmail(session.getEmailAddress())
                             .orElseThrow(
                                     () ->
                                             new IpvCallbackException(
                                                     "Email from session does not have a user profile"));
+            var pairwiseSubject =
+                    segmentedFunctionCall(
+                            "calculatePairwiseSubject",
+                            () ->
+                                    ClientSubjectHelper.getSubject(
+                                            userProfile,
+                                            clientRegistry,
+                                            dynamoService,
+                                            configurationService.getInternalSectorUri()));
+            if (errorObject.isPresent()) {
+                var accountInterventionStatus =
+                        getAccountInterventionStatus(pairwiseSubject.getValue());
+                if (configurationService.isAccountInterventionServiceEnabled()) {
```

Comment:
`configurationService.isAccountInterventionServiceEnabled()` has been removed in this unmerged PR: https://github.com/govuk-one-login/authentication-api/pull/3698